certified-connectors/KyndrylMainframe/apiDefinition.swagger.json

```diff
@@ -0,0 +1,740 @@
+            "CA_POSTCODE": "string",
+            "CA_NUM_POLICIES": 0,
+            "CA_PHONE_MOBILE": "string",
+            "CA_PHONE_HOME": "string",
+            "CA_EMAIL_ADDRESS": "string",
+            "CA_POLICY_DATA": "string"
+          }
+        }
+      }
+    },
+    "putCustomerdetail_response_200": {
+      "type": "object",
+      "properties": {
+        "LGCMAREA": {
+          "type": "object",
+          "properties": {
+            "CA_REQUEST_ID": {
+              "type": "string",
+              "maxLength": 6
+            },
+            "CA_RETURN_CODE": {
+              "type": "integer",
+              "minimum": 0,
+              "maximum": 99
+            },
+            "CA_CUSTOMER_NUM": {
+              "type": "integer",
+              "minimum": 0,
+              "maximum": 9999999999
+            },
+            "CA_REQUEST_SPECIFIC": {
+              "type": "string",
+              "maxLength": 32482
+            },
+            "CA_CUSTOMER_REQUEST": {
+              "type": "object",
+              "properties": {
+                "CA_FIRST_NAME": {
+                  "type": "string",
+                  "maxLength": 10
+                },
+                "CA_LAST_NAME": {
+                  "type": "string",
+                  "maxLength": 20
+                },
+                "CA_DOB": {
+                  "type": "string",
+                  "maxLength": 10
+                },
+                "CA_HOUSE_NAME": {
+                  "type": "string",
+                  "maxLength": 20
+                },
+                "CA_HOUSE_NUM": {
+                  "type": "string",
+                  "maxLength": 4
+                },
+                "CA_POSTCODE": {
+                  "type": "string",
+                  "maxLength": 8
+                },
+                "CA_NUM_POLICIES": {
+                  "type": "integer",
+                  "minimum": 0,
+                  "maximum": 999
+                },
+                "CA_PHONE_MOBILE": {
+                  "type": "string",
+                  "maxLength": 20
+                },
+                "CA_PHONE_HOME": {
+                  "type": "string",
+                  "maxLength": 20
+                },
+                "CA_EMAIL_ADDRESS": {
+                  "type": "string",
+                  "maxLength": 100
+                },
+                "CA_POLICY_DATA": {
+                  "type": "string",
+                  "maxLength": 32267
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {
+    "API Key": {
+      "type": "apiKey",
+      "in": "header",
+      "name": "Ocp-Apim-Subscription-Key"
+    }
+  },
+  "security": [
+    {
+      "API Key": []
+    }
+  ],
+  "tags": [], 
+"x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "www.kyndryl.com"
```

Comment:
@RTkyndryl The 'url' property must be a valid absolute URL. Please change it to "https://www.kyndryl.com"